### ISSUE TITLE
Material Design Lite (MDL) Support

### ISSUE THREAD

**Issue by leocaseiro** [mike-schultz/materialette#22]
Hi, thanks for this awesome app.

That would be nice, if we could use copy the class for the [material design lite](https://getmdl.io/styles/index.html) as well.

Such as `.mdl-color--blue-300` or `.mdl-color-text--blue-300`

You can see more here http://blog.jonathanargentiero.com/material-design-lite-color-classes-list

It should be easy to implement that. Something like ".mdl-color--{color}-{number}" and ".mdl-color-text--{color}-{number}"

My suggest for the menu would be "MDL" and "MDL Text"

I'm happy to give try if you are interested on that.

**Comment by mike-schultz** [comment]
Yeah, that could be useful. Go ahead! I'll probably end up changing the menu with arrows on top to be a drop down to start supporting more modes.

**Comment by oliyy** [comment]
This would actually be pretty nice
